gui/src/components/maintenance/auditlog.vue

```diff
@@ -73,12 +73,64 @@
     <div class="ma-3 mt-5" v-if="showLog">
       <span class="cortx-text-bold cortx-text-lg" id="auditlogtext">{{ $t("maintenance.logs") }}</span>
       <v-divider class="my-2"></v-divider>
-      <span
+      <!-- <span
         class="mb-1 d-block"
         v-for="(log, index) in showLog"
         :key="index"
         id="auditlog-data"
-      >{{ log }}</span>
+      >{{ log }}</span> -->
+      <v-data-table
+            calculate-widths
+            :items="auditLogList"
+            item-key="name"
+            class="cortx-table"
+            id="auditLog-datatable"
+            :hide-default-header="true"
+            :hide-default-footer="true"
+            :disable-pagination="true"
+          >
+            <template v-slot:header="{}">
+              <tr id="auditLog-tableheading">
+                <th
+                  v-for="header in auditLogTableHeaderList"
+                  :key="header.text"
+                  class="tableheader"
+                  id="table-name"
+                >
+                  <span>{{ header.text }}</span>
+                </th>
+              </tr>
+            </template>
+
+            <template v-slot:item="props">
```

Comment:
you can use <template v-slot:item.timestamp="props"> to customize single column instead of whole row.